app/templates/suppliers/details.html

```diff
@@ -5,20 +5,21 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {{ govukBreadcrumbs({
+    "items": [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
+      },
+      {
```

Comment:
If possible we should insert a breadcrumb here in the case where the user is currently applying to a framework

Reply:
@lfdebrux How about?
```
  {% with items = [
      {
        "text": "Digital Marketplace",
        "href": "/"
      },
      {
        "text": "Your account",
        "href": url_for('.dashboard')
      },
      {
        "text": "Apply to " + framework.name,
        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
      },
      {
        "text": service_data.lotName,
        "href": url_for(".framework_submission_lots", framework_slug=framework.slug)
      },
      {
        "text": "Services",
        "href": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot)
      }
    ]
  %}
    {% if one_service_limit %}
      {{ govukBreadcrumbs(items.append({"text": section.name})) }}
    {% else %}
      {{ govukBreadcrumbs(items.extend([
          {
            "text": service_data.get('serviceName', service_data['lotName']),
            "href": url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id)
          },
          {
            "text": section.name
          }
        ]
      )) }}
    {% endif %}
  {% endwith %}
```

Reply:
Does it work? 😆